### PR TITLE
update network detail

### PR DESCRIPTION
**Pull request by mason5991**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,10 +24,50 @@ All contracts on Chainlink must be funded using the local utility token LINK.
...
+### Ethereum Mainnet (Chain ID: 1)
 
-- indexName: cd7bea47b73a4d47a5cedecf00b9b561
-- indexValue: d06b539cdbeb450aa72f2e70223078af
-- aDayChange: 3d2a6108056644c38f98d755536eb5ac
-- aMonthChange: b871377caf1f4ccb851189f8d906c277
+- Payment amount: 0.01 LINK
+- Oracle address: 0xfE2dD37BC29f5fc4E0cad8F58F4Dbf4AddD5A59A
+- LINK token address: 0x514910771af9ca656af840dff83e8264ecf986ca
+- Job ID(NFT Index): eb4f0d822a4c481ca88e5a3ab2fec562
```
Comment (JGJP) [inline README.md:32]: Do we only have one job ID now?

File: README.md [hunk 2]
```diff
@@ -24,10 +24,50 @@ All contracts on Chainlink must be funded using the local utility token LINK.
...
+
+- Payment amount: 0.01 LINK
+- Oracle address: 0xA96474C1A08374EFd0F3C9BC7153FDA7A6c8d9e1
+- LINK token address: 0xb0897686c545045afc77cf20ec7a532e3120e0f1
+- Job ID(NFT Index): eb4f0d822a4c481ca88e5a3ab2fec562
+
+**Important**
+**The LINK provided by the [Polygon (Matic) Bridge](https://wallet.polygon.technology/bridge) is not ERC-677 compatible, so cannot be used with Chainlink oracles. However, it can be [converted to the official LINK token on Polygon (Matic) using Chainlink's PegSwap service](https://pegswap.chain.link/?_ga=2.130074441.756683581.1650345653-1161789045.1649048909)**
+
+### Mumbai Testnet (Chain ID: 80001)
```
Comment (JGJP) [inline README.md:68]: Specify Polygon?

File: README.md (hunk 1 above)
Comment (mason5991) [reply README.md:32]: @JGJP yes, the job will return those 4 parameters together. You can check similar example here
https://github.com/truflation/quickstart/blob/nft-index/NftIndexTester.sol#L48-L57

File: README.md (hunk 2 above)
Comment (mason5991) [reply README.md:68]: Only for polygon mainnet.

Comment (mason5991) [reply README.md]: Renamed to Mumbai (Polygon) Testnet as well

**Pull request merged**